ckanext/datavic_iar_theme/templates/package/snippets/featured_resource.html

```diff
@@ -0,0 +1,24 @@
+{# Featured Resource Preview
+
+    pkg: the package dict
+    
+    #}
+<section class="datapreview">
+    <h3>{{ _('Data preview') }}</h3>
+<div class="resource-view">
+    {% set resource_view = h.featured_resource_preview(pkg.resources) %}
+    {% if resource_view %}
+
+        {% snippet 'package/snippets/resource_view.html',
+            resource_view=resource_view['preview'],
+            resource=resource_view['resource'],
+            package=pkg
+            %}
+    {% else %}
+      {# Views not created #}
+      <div class="data-viewer-info">
+        <p>{{ _("There are no views created for this resource yet.") }}</p>
```

Comment:
This label is misleading. It's shown on the dataset's page and it's not clear, which resource you are talking about, saying "There are no views created for **this** resource yet."